setup.py

```diff
@@ -17,6 +17,7 @@
     include_package_data=True,
     platforms="any",
     install_requires=list(val.strip() for val in open("requirements.txt")),
```

Comment:
Side note: It's not good practice to set the production requirements the same way as the development requirements. I recommend pinning the development requirements strictly in requirements files (or an optional list in pyproject.toml) and have as forgiving pinning for the production as possible in requirements files or directly in setup.py (or pyproject.toml).

It would also be good to change to pyproject.toml at some point.